corehq/apps/style/decorators.py

```diff
@@ -1,27 +1,4 @@
 from functools import wraps
-from corehq.apps.style.utils import set_bootstrap_version3
-from crispy_forms.utils import set_template_pack
-
-
-def use_bootstrap3(view_func):
```

Comment:
then we'd have additional debt to clean @use_bootstrap3 up after and it may always linger around